pnpm-workspace.yaml

```
@@ -0,0 +1,2 @@
+packages:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)